tests/integration/steps_submit_equivocation_proposal.go

```diff
@@ -2,6 +2,46 @@ package main
 
 import "time"
 
+func stepsRejectEquivocationProposal(consumerName string, propNumber uint) []Step {
```

Comment:
Can we add a comment explaining that the prop is rejected because no slash log exists

Reply:
Done.